lib/app/presentation/user/register_page/register_page.dart

```diff
@@ -0,0 +1,145 @@
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_lerning_with_rest_api/app/presentation/user/register_ctrl/register_page_ctrl.dart';
+import 'package:get/get.dart';
+
+import '../../../core/themes/text_styles.dart';
+import '../../../core/values/app_colors.dart';
+import '../../../core/values/app_diemen.dart';
+import '../../../core/values/app_strings.dart';
+import '../../../global_widgets/k_button.dart';
+import '../../../global_widgets/neumorfic_text_field.dart';
+
+class RegisterPage extends StatelessWidget {
+  RegisterPage({Key? key}) : super(key: key);
+  var ctrl = Get.put(RegisterController());
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        backgroundColor: AppColors.colorApp,
+        title: Text(
+          "Register",
+          style: TextStyles.sfProDisplayRegular(
+              AppColors.colorWhite, AppDimen.unitHeight * 10),
+        ),
+      ),
+      body: SingleChildScrollView(
+        child: Padding(
+          padding: const EdgeInsets.all(10.0),
+          child: Column(
+            children: [
+              SizedBox(
+                height: AppDimen.unitHeight * 10,
+              ),
+              Text(
+                AppStrings.appName,
+                style: TextStyles.modernistBold(
+                  AppColors.colorApp,
+                  AppDimen.unitWidth * 30,
+                ),
+              ),
+              SizedBox(
+                height: AppDimen.unitHeight * 20,
+              ),
+              Align(
+                alignment: Alignment.centerLeft,
+                child: Padding(
+                  padding: const EdgeInsets.only(left: 20.0),
+                  child: Text(
+                    "Please enter the detail below to continue.",
+                    style: TextStyles.modernistBold(
+                      AppColors.colorApp,
+                      AppDimen.unitWidth * 15,
+                    ),
+                  ),
+                ),
+              ),
+              Padding(
```

Comment:
This view is repeated on line no. 71, 85, 99, 111.
It would be better to make it as a separate widget and pass the required fields to that widget.

Reply:
Similarly like you have created KTextWidget in this applciation

Reply:
changes done.